spec/features/account_reset/delete_account_spec.rb

```diff
@@ -158,9 +189,9 @@
       click_link t('two_factor_authentication.login_options_link_text')
 
       # Visiting account reset directly should redirect to 2FA
```

Comment:
It's not really impacted by your change, but this comment strikes me as being at odds with what we're asserting after visiting the path. It reads to me as if it's expected to _prevent_ someone from navigating directly there, but we're asserting that they successfully reach the page.

Historically, it looks like this restriction was added in #2421 and then removed in #3389.

I might suggest we could remove the entire context block "as an IAL2 user" altogether. What do you think?

Reply:
I agree ur right, I dont think this makes sense. updating.